package.json

```diff
@@ -7,12 +7,21 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "tsc --noEmit && eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
-    "harpstrata": "^4.0.1",
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-navigation/drawer": "^5.7.5",
+    "@react-navigation/native": "^5.4.0",
+    "@react-navigation/stack": "^5.3.7",
```

Comment:
I think this and possibly some other dependencies can go

Reply:
Removed stack. I don't want to risk breaking it before I have a good tag down so I'm not going to delete any more. The dev environment has proven complex enough that it doesn't always fail immediately when you make a breaking change. Cached resources and such. So I'll wait before removing more.